Extension/package.json

```diff
@@ -5786,6 +5788,1589 @@
+                                            {
+                                                "type": "boolean",
+                                                "description": "%c_cpp.debuggers.logging.engineLogging.description%",
+                                                "default": false
+                                            }
+                                        ]
+                                    },
+                                    "trace": {
+                                        "type": "boolean",
+                                        "description": "%c_cpp.debuggers.logging.trace.description%",
+                                        "default": false
+                                    },
+                                    "traceResponse": {
+                                        "type": "boolean",
+                                        "description": "%c_cpp.debuggers.logging.traceResponse.description%",
+                                        "default": false
+                                    },
+                                    "natvisDiagnostics": {
+                                        "anyOf": [
+                                            {
+                                                "type": "string",
+                                                "enum": [
+                                                    "verbose",
+                                                    "warning",
+                                                    "error",
+                                                    "none"
+                                                ],
+                                                "enumDescriptions": [
+                                                    "%c_cpp.debuggers.logging.category.verbose.description%",
+                                                    "%c_cpp.debuggers.logging.category.warning.description%",
+                                                    "%c_cpp.debuggers.logging.category.error.description%",
+                                                    "%c_cpp.debuggers.logging.category.none.description%"
+                                                ],
+                                                "description": "%c_cpp.debuggers.logging.natvisDiagnostics.description%",
+                                                "default": "none"
+                                            },
+                                            {
+                                                "type": "boolean",
+                                                "description": "%c_cpp.debuggers.logging.natvisDiagnostics.description%",
+                                                "default": false
+                                            }
+                                        ]
+                                    }
+                                }
+                            },
+                            "deploySteps": {
```

Comment:
Is `deploySteps` supported?

Reply:
I would think so - it's handled the during `resolveDebugConfigurationWithSubstitutedVariables` - regardless of the type of debugger .